utilities/rpp-performancetests/OCL_NEW/BatchPD_ocl_pkd3.cpp

```diff
@@ -3369,6 +3312,41 @@ int main(int argc, char **argv)
 
             break;
         }
+        case 75:
+        {
+            test_case_name = "fast_corner_detector";
+            missingFuncFlag = 1;
```

Comment:
There are lots of missing functions flag. Are those expected? 
We need to print out error code saying not implemented for those cases